.github/workflows/auto-update-linters.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       ###################################
       - name: Build image
         shell: bash
-        run: docker build --build-arg "BUILD_DATE=${BUILD_DATE}" --build-arg "BUILD_REVISION=auto_update_${GITHUB_SHA}" --build-arg "BUILD_VERSION=auto_update_${GITHUB_SHA}" --no-cache -t oxsecurity/megalinter:auto_update_${GITHUB_SHA} . | while read line ; do echo "$(date +'%H:%M:%S')| $line"; done;
+        run: DOCKER_BUILDKIT=1 docker build --build-arg "BUILD_DATE=${BUILD_DATE}" --build-arg "BUILD_REVISION=auto_update_${GITHUB_SHA}" --build-arg "BUILD_VERSION=auto_update_${GITHUB_SHA}" --no-cache -t oxsecurity/megalinter:auto_update_${GITHUB_SHA} . | while read line ; do echo "$(date +'%H:%M:%S')| $line"; done;
```

Comment:
Nice! It will be useful, since there are some more newer tricks that we can use in Dockerfiles if we also use an external front end (see https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/reference.md , especially `COPY --link`)

Is there some differences of using this rather than `docker buildx build`?

Reply:
I have no idea , I don't know buildx 🤣 
Please let me know if you do :)

Reply:
I couldn't find the answer I was looking for in 15 min of search, so we'll just change it again if needed